routes/inertia.php

```diff
@@ -57,4 +65,10 @@
                         ->name('team-invitations.destroy');
         }
     });
+
+    // Socialite...
+    if (Jetstream::hasSocialiteFeatures()) {
+        Route::get('/socialite/{provider}', [SocialiteController::class, 'redirectToProvider'])->name('socialite.redirect');
```

Comment:
If you loop `socialite.providers` to define the routes it does a few things:

- makes a cleaner use of route helpers
 ```
route('socialite.github')
```
```
route('socialite.redirect', ['provider' => 'github'])
```

You can set the parameter like so: `Route::get(.., ...)->defaults('provider', $provider);`

- It also 404s instead of 500s from passing a unknown provider to `Socialite::driver($anythingGoesHere)`

Reply:
You can still pass in parameters to the `route` helper, for example, `route('socialite.redirect', 'github')`.